### ISSUE TITLE
Input group button addons are not styled differently in the success, warning, and error validation states

### ISSUE THREAD

**Issue by dtrebbien** [twbs/bootstrap#17422]
Unlike text addons, an input group button addon is not styled according to the form group's validation state.  I think that at a minimum, the border color of the button addon should be changed to the green/yellow/red color of the validation state.

Here is a JSFiddle demonstration:
http://jsfiddle.net/7hyjwnvy/4/

I am using Bootstrap 3.3.5 and jQuery 2.1.4.  Tested Google Chrome 44.0.2403.157 on Win8.1 and Safari 8.0.8 and Firefox 40.0.3 on Mac OS 10.10.5.

**Comment by twbs-lmvtfy** [comment]
Hi @dtrebbien!

You appear to have posted a live example (http://fiddle.jshell.net/7hyjwnvy/3/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 63, column 17: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 96, column 17: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 129, column 17: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by dtrebbien** [comment]
http://jsfiddle.net/7hyjwnvy/4/

**Comment by cvrebert** [comment]
Duplicate of #11659.
Changing just the border color was specifically rejected in https://github.com/twbs/bootstrap/pull/12740#issuecomment-35121574
